### PR TITLE
Use Python's 'or', not '||'

### LINKED ISSUE #89: Invalid syntax
#### Issue Summary
Even the syntax highlighting here on GitHub (in red) shows this to be invalid syntax. Simply changing `||` to `or` should solve this problem. One other consideration could be to use [`self.assertTrue(...)`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue) instead of `self.assertEqual(True, ...)` for the other tests in this file.

@mbernier -> b26031be27a36460758259e74b4b372a8b2ff07d

https://github.com/sendgrid/smtpapi-python/blob/97cca3460802d62d288c06353d4f687d3c0e7ed6/test/test_project.py#L12

https://github.com/sendgrid/smtpapi-python/blob/97cca3460802d62d288c06353d4f687d3c0e7ed6/test/test_project.py#L16

https://github.com/sendgrid/smtpapi-python/blob/97cca3460802d62d288c06353d4f687d3c0e7ed6/test/test_project.py#L52

#### Steps to Reproduce

```python
$ python2
Python 2.7.15rc1 (default, Apr 15 2018, 21:51:34) 
[GCC 7.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> os.path.isdir("./Dockerfile") || os.path.isdir("./docker/Dockerfile")
  File "<stdin>", line 1
    os.path.isdir("./Dockerfile") || os.path.isdir("./docker/Dockerfile")
                                   ^
SyntaxError: invalid syntax

$ python
Python 3.7.1 (default, Oct 20 2018, 18:18:46) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> os.path.isdir("./Dockerfile") || os.path.isdir("./docker/Dockerfile")
  File "<stdin>", line 1
    os.path.isdir("./Dockerfile") || os.path.isdir("./docker/Dockerfile")
                                   ^
SyntaxError: invalid syntax

```

### PR DESCRIPTION
**Pull request by hugovk**
# Fixes #89 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation about the functionality in the appropriate .md file
- [x] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Python uses `or`, not `||`

### CODE REVIEW

File: test/test_project.py [hunk 1]
```diff
@@ -9,11 +9,11 @@ class ProjectTests(unittest.TestCase):
 
     # ./Docker or docker/Docker
     def test_docker_dir(self):
-        self.assertEqual(True, os.path.isdir("./Dockerfile") || os.path.isdir("./docker/Dockerfile"))
+        self.assertEqual(True, os.path.isdir("./Dockerfile") or os.path.isdir("./docker/Dockerfile"))
```
Comment (42B) [inline test/test_project.py]: `self.assertTrue`

File: test/test_project.py [hunk 2]
```diff
@@ -9,11 +9,11 @@ class ProjectTests(unittest.TestCase):
...
 
     # ./docker-compose.yml or ./docker/docker-compose.yml
     def test_docker_compose(self):
-        self.assertEqual(True, os.path.isfile('./docker-compose.yml') || os.path.isfile('./docker/docker-compose.yml'))
+        self.assertEqual(True, os.path.isfile('./docker-compose.yml') or os.path.isfile('./docker/docker-compose.yml'))
```
Comment (42B) [inline test/test_project.py]: `self.assertTrue`

File: test/test_project.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ def test_issue_template(self):
 
     # ./LICENSE.md
     def test_license(self):
-        self.assertEqual(True, os.path.isfile('./LICENSE.md') || os.path.isfile('./LICENSE.txt'))
+        self.assertEqual(True, os.path.isfile('./LICENSE.md') or os.path.isfile('./LICENSE.txt'))
```
Comment (42B) [inline test/test_project.py]: `self.assertTrue`

**Comment by misterdorm** [comment]
Closing in favor of #86.

**Pull request closed without merging**